Blizzard.IO.RabbitMQ/Blizzard.IO.RabbitMQ.csproj

```diff
@@ -8,4 +8,8 @@
     <ProjectReference Include="..\Blizzard.IO.Core\Blizzard.IO.Core\Blizzard.IO.Core.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Implementaions\" />
+  </ItemGroup>
```

Comment:
Why only this folder is in the csproj?

Reply:
i dont know why the interfaces folder isn't here as well